iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -537,24 +537,38 @@ private static ValidationLevel checkPhaseTapChangerRegulation(Validable validabl
+    }
+
+    private static ValidationLevel checkRegulatingPhaseTapChanger(Validable validable,
+            PhaseTapChanger.RegulationMode regulationMode,
+            double regulationValue, Terminal regulationTerminal, boolean loadTapChangingCapabilities,
+            ActionOnError actionOnError, ReportNode reportNode) {
+        ValidationLevel validationLevel = ValidationLevel.STEADY_STATE_HYPOTHESIS;
+        if (!loadTapChangingCapabilities) {
+            throwExceptionOrLogError(validable, "regulation cannot be enabled on phase tap changer without load tap changing capabilities", actionOnError,
                         id -> NetworkReports.ptcPhaseRegulationCannotBeEnabledWithoutLoadTapChanging(reportNode, id));
-                validationLevel = ValidationLevel.min(validationLevel, ValidationLevel.EQUIPMENT);
-            }
-            if (Double.isNaN(regulationValue)) {
-                throwExceptionOrLogError(validable, "phase regulation is on and threshold/setpoint value is not set", actionOnError,
+            validationLevel = ValidationLevel.min(validationLevel, ValidationLevel.EQUIPMENT);
+        }
+        if (Double.isNaN(regulationValue)) {
+            throwExceptionOrLogError(validable, "phase regulation is on and threshold/setpoint value is not set", actionOnError,
                         id -> NetworkReports.ptcPhaseRegulationRegulationValueNotSet(reportNode, id));
-                validationLevel = ValidationLevel.min(validationLevel, ValidationLevel.EQUIPMENT);
-            }
-            if (regulationTerminal == null) {
-                throwExceptionOrLogError(validable, "phase regulation is on and regulated terminal is not set", actionOnError,
+            validationLevel = ValidationLevel.min(validationLevel, ValidationLevel.EQUIPMENT);
+        }
+        if (regulationTerminal == null) {
+            throwExceptionOrLogError(validable, "phase regulation is on and regulated terminal is not set", actionOnError,
                         id -> NetworkReports.ptcPhaseRegulationNoRegulatedTerminal(reportNode, id));
-                validationLevel = ValidationLevel.min(validationLevel, ValidationLevel.EQUIPMENT);
-            }
+            validationLevel = ValidationLevel.min(validationLevel, ValidationLevel.EQUIPMENT);
         }
-        if (regulationTerminal != null && regulationTerminal.getVoltageLevel().getNetwork() != network) {
-            throw new ValidationException(validable, "phase regulation terminal is not part of the network");
+        if (regulationMode == PhaseTapChanger.RegulationMode.CURRENT_LIMITER && regulationValue < 0) {
+            throwExceptionOrLogError(validable, "phase tap changer in CURRENT_LIMITER mode must have a non-negative regulation value", actionOnError,
+                    id -> NetworkReports.ptcPhaseRegulationModeAndValueNotCoherent(reportNode, id));
+            validationLevel = ValidationLevel.min(validationLevel, ValidationLevel.EQUIPMENT);
```

Comment:
Always throw an exception:
```suggestion
            throw new ValidationException(validable, "phase tap changer in CURRENT_LIMITER mode must have a non-negative regulation value");
```